src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14225,89 +14225,1275 @@ public static void Main()
 5").VerifyDiagnostics();
         }
 
-        [Fact]
-        public void LambdaDefaultParameter_ArrayCommonType_DefaultValueMismatch()
+        private static readonly string s_anonymousDelegateWithDecimalConstant = $$"""
```

Comment:
With these types of tests, it's good to have a comment above with pseudo-code for what the C# equivalent would be, for ease of review and later reading.

Reply:
Thanks; it was probably more obvious when there was a C# equivalent next to them, but you're right, it's not anymore. Added comments.